app/lib/mappers/mapOpeningTimes.js

```diff
@@ -0,0 +1,29 @@
+function getSession(session) {
+  return {
+    opens: session['s:fromTime'],
+    closes: session['s:toTime'],
+  };
+}
+
+function getSessions(session) {
+  const timesSessions = session['s:timesSession'];
+  if (timesSessions.constructor === Array) {
+    return timesSessions.map(getSession);
+  }
+  if (timesSessions === 'Closed') {
+    return [];
+  }
+  return [
+    getSession(timesSessions),
+  ];
+}
+
+function mapOpeningTimes(rawTimes) {
+  const openingTimes = {};
+  rawTimes['s:daysOfWeek']['s:dayOfWeek'].forEach((dayOfWeek) => {
+    openingTimes[dayOfWeek['s:dayName'].toLowerCase()] = getSessions(dayOfWeek['s:timesSessions']);
```

Comment:
Will either `dayOfWeek['s:dayName']` or `dayOfWeek['s:timesSessions']` ever be `null`/`undefined`?